Packs/Bonusly/Integrations/Bonusly/Bonusly.yml

```diff
@@ -0,0 +1,659 @@
+category: Utilities
+commonfields:
+  id: Bonusly
+  version: -1
+configuration:
+- defaultvalue: https://bonus.ly/api/v1/
+  display: Server URL (e.g. https://bonus.ly/api/v1/)
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  name: api_key
+  required: true
+  type: 4
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Enter the name of the specific bonusly application if required.
+  hidden: false
+  name: application_name
+  required: false
+  type: 0
+- defaultvalue: '20'
+  display: Limit the number of bonuses to fetch (Default 20)
+  hidden: false
+  name: limit
+  required: false
+  type: 0
+- display: Filter to receive only bonuses that match either a giver or receiver with
+    this email. (person@somewhere.com)
+  hidden: false
+  name: user_email
+  required: false
+  type: 0
+- display: Filter based on a hashtag like %23teamwork
+  hidden: false
+  name: hashtag
+  required: false
+  type: 0
+- defaultvalue: '1'
+  display: Start fetching from X days ago (defaults to 1 day)
+  hidden: false
+  name: fetch_time
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+description: This is the Hello World integration for getting started.
```

Comment:
please change the description

Reply:
Thanks for pointing this out much appreciated!